src/Components/MovieInfo/MovieInfo.js

```diff
@@ -40,6 +40,13 @@ class MovieInfo extends React.Component {
     return (
       <main>
         <Header />
+
+        <section className="indv-movie">
+          <h1 className="title" data-testid="movie-title">{this.state.title}</h1>
```

Comment:
Can you query this element by its role of "header" instead of the test id? Try and keep `testids` as a last resort for querying.